helm/keygen/templates/job.yaml

```diff
@@ -101,13 +101,11 @@ spec:
                 name: {{ . }}
             {{- end }}
             {{- end }}
-            {{- if .Values.softHsmCM }}
```

Comment:
cant remove this..

atleast if needed to be removed add it to extraEnvCM

Reply:
and even after that need to assign the values dynamically.

Reply:
added back